pkg/oc/cli/cmd/login/loginoptions.go

```diff
@@ -317,15 +317,19 @@ func (o *LoginOptions) gatherProjectInfo() error {
 		}
 		o.Project = current.Name
 
-		fmt.Fprintf(o.Out, "You have access to the following projects and can switch between them with '%s project <projectname>':\n\n", o.CommandName)
-		for _, p := range projects.List() {
-			if o.Project == p {
-				fmt.Fprintf(o.Out, "  * %s\n", p)
-			} else {
-				fmt.Fprintf(o.Out, "    %s\n", p)
+		if len(projectsItems) > 50 {
```

Comment:
Please make the `50` a constant `maxProjectItemLimit` (or a similar name) at the top of this file.